setup.cfg

```diff
@@ -47,6 +47,8 @@ markers=
 filterwarnings=
     error
     ignore::pytest.PytestUnraisableExceptionWarning
+    ignore::RuntimeWarning:desc.compute
```

Comment:
do we want this change? I thought that Rory wanted to catch warnings like this in the tests, then change the underlying code to avoid it, like in this line: https://github.com/PlasmaControl/DESC/blob/a92c2f9f9822dc843faa017ca042555d86149e48/desc/grid.py#L206

Reply:
I think it's fine. With the axis stuff there are too many individual places where we divide by zero and then overwrite with the correct value, so ignoring the warning for just the compute module seems reasonable.